### PR TITLE
Cockpit 291

### PR DESCRIPTION
**Pull request by garrett**
Preview: https://garrett.github.io/cockpit-project.github.io/blog/cockpit-291.html

### CODE REVIEW

**Comment by garrett** [comment]
I haven't removed tags or edited anything yet.

File: _posts/2023-05-02-cockpit-291.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 291, cockpit-machines 289, and cockpit-podman 68:
+
+
+## Update to patternfly 5 alpha
+
+State: **merged** @ <https://github.com/cockpit-project/cockpit/pull/18672>
+{:.note}
```
Comment (KKoukiou) [inline _posts/2023-05-02-cockpit-291.md:20]: Please link to blog post https://cockpit-project.org/blog/patternfly-v5-alpha.html

**Review by martinpitt — CHANGES_REQUESTED** [review]
Cheers!

File: _posts/2023-05-03-cockpit-291.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+title: Cockpit 291
+author: garrett
+date: '2023-05-03'
+tags: cockpit, machines
```
[the remark this answers is not in the document]
Comment (garrett) [reply _posts/2023-05-03-cockpit-291.md:5]:
```suggestion
tags: cockpit, machines, podman
```

File: _posts/2023-05-03-cockpit-291.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+Here are the release notes from cockpit 291, cockpit-machines 289.1, and cockpit-podman 68:
+
+
+## Update to patternfly 5 alpha
+
+Cockpit is now an "early adopter" of PatternFly 5. Everything should look the same. We ensured all functional and visual tests pass. While this should not affect people using Cockpit, it definitely will affect everyone developing for Cockpit. Read more on [our PatternFly 5 blog post](https://cockpit-project.org/blog/patternfly-v5-alpha.html).
+
+## Option to use RAW volume during VM creation
+
+Users can now choose RAW as the storage format for the primary disk of their VM during the VM creation process.
```
[the remark this answers is not in the document]
Comment (garrett) [reply _posts/2023-05-03-cockpit-291.md:23]: "RAW" is not an acronym. It should be "raw". Also: We should not refer to people as "users". I'll reword this section.

File: _posts/2023-05-03-cockpit-291.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+![Screenshot from 2023-05-03 12-32-48]({{ site.baseurl }}/images/291-option-to-use-raw-volume-during-vm-creation.png)
+
+## Redesign VM's CPU configuration
+
+The CPU model dialog and vCPU topology have been combined into a single field and a dialog. Additionally, a few minor improvements have been added, including vCPU maximum and vCPU count now being number inputs. Furthermore, the dialog layout has been updated to a vertical form.
+
+![Screenshot from 2023-05-03 12-32-03]({{ site.baseurl }}/images/291-redesign-vm-s-cpu-configuration.png)
+![Screenshot from 2023-05-03 12-40-15]({{ site.baseurl }}/images/291-redesign-vm-s-cpu-configuration-2.png)
```
[the remark this answers is not in the document]
Comment (garrett) [reply _posts/2023-05-03-cockpit-291.md:32]: People writing the descriptions need to have a blank line between images. I usually fix this myself.

[the remark this answers is not in the document]
Comment (garrett) [reply _posts/2023-05-03-cockpit-291.md:32]:
```suggestion

![Screenshot from 2023-05-03 12-40-15]({{ site.baseurl }}/images/291-redesign-vm-s-cpu-configuration-2.png)
```

File: _posts/2023-05-03-cockpit-291.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
+---
+title: Cockpit 291
+author: garrett
+date: '2023-05-03'
+tags: cockpit, machines
+slug: cockpit-291
+category: release
+summary: ''
```
[the remark this answers is not in the document]
Comment (garrett) [reply _posts/2023-05-03-cockpit-291.md:8]:
```suggestion
summary: 'PatternFly 5, create VMs with raw disks, VM CPU editing refinements'
```

**Review by martinpitt — APPROVED** [review]
Yay, thanks! LGTM

**Comment by garrett** [comment]
Oh, I still haven't edited the text yet.

File: _posts/2023-05-03-cockpit-291.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+## Machines: Option to use RAW volume during VM creation
+
+Users can now choose RAW as the storage format for the primary disk during the VM creation process.
+
+![Screenshot from 2023-05-03 12-32-48]({{ site.baseurl }}/images/291-option-to-use-raw-volume-during-vm-creation.png)
+
+## Machines: Redesign VM's CPU configuration
+
+The CPU model dialog and vCPU topology have been combined into a single field and a dialog. Additionally, a few minor improvements have been added, including vCPU maximum and vCPU count now being number inputs. Furthermore, the dialog layout has been updated to a vertical form.
```
Comment (garrett) [inline _posts/2023-05-03-cockpit-291.md:29]:
```suggestion
## Machines: Redesign virtual machine CPU configuration

The CPU model dialog and vCPU topology have been merged into a single overview entry and dialog. Additionally, a few minor improvements have been added, including vCPU maximum and vCPU count now being number inputs.
```

File: _posts/2023-05-03-cockpit-291.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+Here are the release notes from cockpit 291, cockpit-machines 289.1, and cockpit-podman 68:
+
+
+## Update to patternfly 5 alpha
+
+Cockpit is now an "early adopter" of PatternFly 5. Everything should look the same. We ensured all functional and visual tests pass. While this should not affect people using Cockpit, it definitely will affect everyone developing for Cockpit. Read more on [our PatternFly 5 blog post](https://cockpit-project.org/blog/patternfly-v5-alpha.html).
+
+## Machines: Option to use RAW volume during VM creation
+
+Users can now choose RAW as the storage format for the primary disk during the VM creation process.
```
Comment (garrett) [inline _posts/2023-05-03-cockpit-291.md:23]:
```suggestion
## Machines: VM creation supports raw disk format

During virtual machine creation, there is now a raw disk option in the storage format selection.
```

File: _posts/2023-05-03-cockpit-291.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from cockpit 291, cockpit-machines 289.1, and cockpit-podman 68:
+
+
+## Update to patternfly 5 alpha
+
+Cockpit is now an "early adopter" of PatternFly 5. Everything should look the same. We ensured all functional and visual tests pass. While this should not affect people using Cockpit, it definitely will affect everyone developing for Cockpit. Read more on [our PatternFly 5 blog post](https://cockpit-project.org/blog/patternfly-v5-alpha.html).
```
Comment (garrett) [inline _posts/2023-05-03-cockpit-291.md:19]:
```suggestion
## Update to PatternFly 5 alpha

Cockpit is now an "early adopter" of PatternFly 5. Everything should look the same. We ensured all functional and visual tests pass.

While this should not affect anyone using Cockpit, it definitely will affect everyone developing for Cockpit. Read more on [our PatternFly 5 blog post](https://cockpit-project.org/blog/patternfly-v5-alpha.html).
```

**Review by garrett — COMMENTED** [review]
I've found the time to review (and change) the text. Apologies for the delay.

**Pull request merged**